website/docs/reference/commands/build.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "build"
+id: "build"
+---
+
+<Changelog>
+
+- Introduced in **v0.21.0**
+    
+</Changelog>
+
+The `dbt build` command will:
+- run models
+- test tests
+- snapshot snapshots
+- seed seeds
+
+In DAG order, for selected resources or an entire project.
```

Comment:
Should we also clarify the command order? IE which of these takes precedence on a given node in the DAG if multiple apply (looks to be Run,Test,Snapshot which is the order it's in now, just might be helpful to explicitly note).

Reply:
In order to break a DAG "tie," it looks like (in local testing) the implicit order is model, seed, snapshot, test. That's alphabetical order (hah), but I'm not sure where that comes from—if anything, I might expect it to be the order defined [here](https://github.com/dbt-labs/dbt/blob/6854e6746406d87e3e800d7e535c85aac5bf6d2e/core/dbt/task/build.py#L23-L29).

In any event, I think users should imagine the order as non-deterministic between independent nodes that can be parallelized, and very much deterministic for dependencies.